packages/stryker/src/reporters/EventRecorderReporter.ts

```diff
@@ -33,9 +33,8 @@ export default class EventRecorderReporter implements StrictReporter {
     return this._baseFolder;
   }
 
-
-  private writeToFile(index: number, methodName: keyof Reporter, data: any) {
-    let filename = path.join(this.baseFolder, `${this.format(index)}-${methodName}.json`);
+  private writeToFile(methodName: keyof Reporter, data: any) {
+    let filename = path.join(this.baseFolder, `${this.format(this.index++)}-${methodName}.json`);
```

Comment:
Are you sure you want index++? I know that we did that in the original code but that was called from another method. I'm not sure if the postincrement in `myMethod(this.index++)` is called before or after the call has been made.